src/app/components/register-card/register-card.component.html

```diff
@@ -0,0 +1,71 @@
+
+        <mat-card-content class="register-card-content">
+              <form class="register-form">
+                
+                <div class="register-card-item">
+                  <mat-icon class="register-item-icon" aria-hidden="false" aria-label="Person icon">person_outline</mat-icon>
+
+                  <mat-form-field class="register-form-item" appearance="fill">
+                    <mat-label>Name</mat-label>
+                    <input matInput type="text" placeholder="Name" [formControl]="registerNameControl">
+                    
+                    <mat-error *ngIf="registerNameControl.hasError('required')">
+                      Name is <strong>required</strong>
+                    </mat-error>
+
+                  </mat-form-field>
+                </div>
+                
+                <div class="register-card-item">
+                  <mat-icon class="register-item-icon" aria-hidden="false" aria-label="Mail">mail</mat-icon>
+
+                  <mat-form-field class="register-form-item" appearance="fill">
+                    <mat-label>Email</mat-label>
+                    <input matInput type="email" placeholder="Email" [formControl]="registerEmailControl">
+                    
+                    <mat-error *ngIf="registerEmailControl.hasError('pattern') && !registerEmailControl.hasError('required')">
+                      Please provide your <strong>firstname.lastname@tss-yonder.com</strong> mail account
+                    </mat-error>
+
+                    <mat-error *ngIf="registerEmailControl.hasError('required')">
+                      Email is <strong>required</strong>
+                    </mat-error>
+
+                    <mat-hint>Register with your @tss-yonder.com mail account</mat-hint>
+                  </mat-form-field>
+                </div>
+
+                <div class="register-card-item">
+                  <mat-icon class="register-item-icon" aria-hidden="false" aria-label="Mail">key</mat-icon>
+
+                  <mat-form-field class="register-form-item" appearance="fill">
+                    <mat-label>Password</mat-label>
+                    <input matInput type="password" placeholder="Password" [formControl]="registerPasswordControl">
+                    
+                    <mat-error *ngIf="registerPasswordControl.hasError('minlength') && !registerPasswordControl.hasError('required')">
+                      Password should be <strong>at least</strong> 6 characters long.
+                    </mat-error>
+
+                    <mat-error *ngIf="registerPasswordControl.hasError('required')">
+                      Password is <strong>required</strong>
+                    </mat-error>
+                    
+                  </mat-form-field>
+                </div>
+
+              </form>
+        </mat-card-content>
+
+        <app-button class="register-button" buttonText="Register" [disabled]="!isRegisterFormDone()" (click)="onClick()"></app-button>
```

Comment:
`onClick` choose better name for functions; if you'd have more buttons on the screen you'll need to know in the .ts to which button the event corresponds to.
`onRegisterUser` would be a better name